apps/gateway/src/routes/v2/conversations/createEvaluationResult/createEvaluationResult.handler.ts

```diff
@@ -122,3 +74,45 @@ async function validatedEvalConnectedToDocument({
     throw new BadRequestError('The evaluation is not connected to this prompt')
   }
 }
+
+// @ts-expect-error: streamSSE has type issues
```

Comment:
is the type issue really because of streamSSE? I think it is because we are separating the handler from the magical createRoute, I would put that in the comment instead of streamSSE? (Also this handler does nothing about SSEs)

Reply:
yeah there's no sse herem, why are we escaping the type checking?